style: remove ANSI color codes from unsupported environments during i…

outerwear opened the pull request:
This commit updates the heading function in the install script to prevent that raw output.

## Current behavior

Currently when installing ockam on environments that don't support ansi color codes (specifically the SGR sequences used here), the raw code is output to the terminal like so:
```
\033[0;33m   GET STARTED:\033[0m
```

## Proposed changes

This simply removes the code from the conditional check to reflect the behavior of other items with color codes.

## Checks

- [x ] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x ] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [ x] There are no Merge commits in this Pull Request. The Ockam repo maintains a linear commit history so we merge PRs by rebasing them to the develop branch. Rebasing to the latest develop branch and force pushing to your PR branch is okay.
- [ x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [ x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

mrinalwadhwa commented:
@outerwear Thank you for spending time on this, while we review please

- [x] Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will make the commit as verified.

outerwear commented:
Just merged myself into contributors. Looks like the linter doesn't like uppercase ANSI in the subject line, I'll adjust that.

mrinalwadhwa commented:
> Just merged myself into contributors.

🤔 I think you missed turning that into a pull request to the `ockam-contributors` repo https://github.com/build-trust/ockam-contributors/pulls

outerwear commented:
> > Just merged myself into contributors.
> 
> 🤔 I think you missed turning that into a pull request to the `ockam-contributors` repo https://github.com/build-trust/ockam-contributors/pulls

:joy: been a while since using github gui for this kind of stuff, updated and sent PR to correct repo

metaclips committed:
style: remove ansi color codes from unsupported environments during install

For environments that don't support ANSI Select Graphic Rendition sequenes,
the heading would still display the raw code like so:
\033[0;33m   GET STARTED:\033[0m
This commit updates the heading function in the install script to prevent that
raw output.

Signed-off-by: outerwear <michaelcoates@protonmail.com>
Co-authored-by: Paulo Zulato <paulozulato@gmail.com>
Co-authored-by: Michael Coates <michaelcoates@protonmail.com>

metaclips commented:
There's a PR that adds same fix https://github.com/build-trust/ockam/pull/5950, I made both PRs co-authors.
